src/components/Button.tsx

```diff
@@ -3,19 +3,18 @@ import React from "react";
 interface ButtonProps {
   text: string;
   onClick?: () => void;
-  type?: "button" | "submit" | "reset";
-  variant?: "primary" | "success";
-};
+  variant?: ButtonVariant;
+}
+
+export enum ButtonVariant {
+  primary = "primary",
+  secondary = "secondary",
+}
 
-export const Button = ({
-  onClick,
-  text,
-  type = "button",
-  variant = "primary",
-}: ButtonProps) => {
-  const getButtonClassName = (variant: string) => {
+export const Button = ({ onClick, text, variant = ButtonVariant.primary }: ButtonProps) => {
+  const getButtonClassName = (variant: ButtonVariant) => {
```

Comment:
remove this function and tailwind. Use styled-component

Reply:
well, i have question here 
If i delete this function, i will not use delete function any more.
did you mean, i should directly create a delete function in Task.tsx instead of generating a component?

Reply:
?